css/listItem.css

```diff
@@ -43,20 +43,18 @@
 }
 
 .searchbar-item i,
+.searchbar-item > .i:first-child,
```

Comment:
This can be removed now that we're using the icon font.

Reply:
Actually, some of the icons are `<i>` elements, and others are `<span class="i">` still, so removing this broke the commands dropdown:

<img width="493" alt="Screen Shot 2020-07-16 at 10 18 22 PM" src="https://user-images.githubusercontent.com/10314059/87744995-691f8180-c7b2-11ea-9ce5-9a42874c83f6.png">

It'd probably be best to pick one of the two methods (I don't think there's any real difference) and use it for all the icons.

Reply:
Made `<span class="i">` -> `<i class="i">`, hope I have covered them all XD